docs/changehistory/NextVersion.md

```diff
@@ -20,6 +20,29 @@ Table of contents:
 
 ## API deprecations
 
+### @itwin/core-bentley
```

Comment:
nit: missing in the ToC above
only update this if you are reacting to the other comment on this file. this will get fixed automatically by the next time someone edits this file and has markdownlint/formatters enabled